front/lib/models/data_source.ts

```diff
@@ -47,6 +53,12 @@ DataSource.init(
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
+    connectedAt: {
+      type: DataTypes.DATE,
+      // TODO(2024-01-25 flav) Set `allowNull` to `false` once backfilled.
```

Comment:
Like webcrawler?

Reply:
Like folders

Reply:
any data source without connectionId

Reply:
Ok but we can't still keep who created them, right?

Reply:
But all that being said... we could have that for folders as well

But we likely want to change the name?

editedByUser / editedAt ?

Reply:
Sorry didn't submit that last message ^

Reply:
Agreed we can record that info for all data sources :+1: 
But connectedAt is maybe not the best naming?

Reply:
I'll go ahead and change the naming. IMHO, it's preferable to enforce stricter values wherever feasible to avoid having to write conditional code later on to handle exceptions 😅.

Reply:
Change the naming to `editedAt` and `editedByUserId`.